ddr/lib/ddr-scanner/dwarf/AixSymbolTableParser.cpp

```diff
@@ -120,7 +122,11 @@ dwarf_init(int fd,
 	/* Call dump to get the symbol table. */
 	if (DW_DLV_OK == ret) {
 		stringstream ss;
+#ifdef J9OS_I5
+		ss << "dump -tvXany " << scanFileName << " 2>&1";
```

Comment:
Hi Keith, sorry for late response. I had a training these days.  
For this comment , I am not quite understand. Do you mean remove the J9OS_I5 check ? In this way, both I5 and AIX would using the same code  ?  I am wondering this would change the behavior of AIX.

Reply:
Yes, I'm suggesting that it should work the same for AIX and I5. If it's done right, it won't change the behavior on AIX.

Reply:
ok. I agree . Would refine code according.